parent/pom.xml

```diff
@@ -1220,6 +1228,12 @@
         <groupId>io.zeebe</groupId>
         <artifactId>zeebe-test-container</artifactId>
         <version>${version.zeebe-test-container}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>io.camunda</groupId>
+            <artifactId>zeebe-client-java</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
❓ Is this because the test container artifact still uses the Zeebe client artifact? If so, this looks like a bad hen-and-egg problem to me since that is a circular dependency, right? The test container project depends on Zeebe and the Zeebe project depends on the test container.

Reply:
Yes, the `zeebe-test-container` depends on `zeebe-client-java:8.4.0`